src/effects/effectchainslot.h

```diff
@@ -35,7 +34,8 @@ class EffectChainSlot : public QObject {
     EffectSlotPointer addEffectSlot(const QString& group);
     EffectSlotPointer getEffectSlot(unsigned int slotNumber);
 
-    void registerInputChannel(const ChannelHandleAndGroup& handle_group);
+    void registerInputChannel(const ChannelHandleAndGroup& handle_group,
+                              const double initialValue = 0.0);
```

Comment:
Adding a new parameter to this function with a default value is a clever way to implement this. :+1: I was thinking that logic should be moved to StandardEffectRack anyway.